src/index.html

```diff
@@ -11,7 +11,115 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div>
+        <fieldset>
+          <legend>Personal information</legend>
+          <div>
+            <label class="form-field" for="surname">Surname:</label>
+            <input type="text" name="surname" id="surname">
+          </div>
+          <div>
+            <label class="form-field" for="name">Name:</label>
+            <input type="text" name="name" id="name">
+          </div>
+          <div>
+            <label class="form-field" for="age">How old are You?</label>
+            <input type="number" name="age" value="12"
+            id="age" min="1" max="99"
+            >
+          </div>
+          <div>
+            <label class="form-field setdown" for="data">Full date of birth:</label>
+            <input type="date" name="data" id="data">
+          </div>
+          <div>
+            <label for="agreemant">
+              I accept the term of the agreement
+            </label>
+            <input type="checkbox" name="agreemant" id="agreemant">
+          </div>
+        </fieldset>
+      </div>
+      <div class="fildset">
+        <fieldset>
+          <legend>Registration</legend>
+          <div>
+            <label class="form-field" for="email">E-mail:</label>
+            <input type="email" name="email" id="email"
+            placeholder="email@example.com"
+          >
+          </div>
+          <div>
+            <label for="password">Password:</label>
+            <input type="password" name="password" id="password">
+          </div>
+        </fieldset>
+      </div>
+      <div class="fildset">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <div class="form-field">
+            Do you love cats?
+            <label>
+              <input type="radio" name="sex" value="yes">
+              Yes
+            </label>
+            <label>
+              <input type="radio" name="sex" value="no">
+              No
+            </label>
+          </div>
+          <div>
+            <label class="form-field" for="color">What is your favorite color?</label>
+            <input type="color" name="color" id="color">
+          </div>
+          <div>
+            <label class="form-field" for="time">What time do you go to bed?</label>
+            <input type="time" name="time" id="time">
+          </div>
+          <div>
+            <label class="form-field" for="car">What are your favorite brand of cars?</label>
```

Comment:
Если получается длинная строка - надо разделять 
```suggestion
            <label class="form-field" for="car">
              What are your favorite brand of cars?
            </label>
```